package.json

```diff
@@ -27,6 +27,7 @@
       "@semantic-release/commit-analyzer",
       "@semantic-release/release-notes-generator",
       "@semantic-release/github",
+      "@semantic-release/npm",
```

Comment:
the `npm` plugin updates the `package.json` file. It's a built-in plugin. It would usually publish the package to the npm reguistry but since we set `"private": true` in our package.json it won't